src/Microsoft.AspNet.Razor/Generator/GeneratedTagHelperContext.cs

```diff
@@ -30,12 +30,13 @@ public GeneratedTagHelperContext()
             ExecutionContextAddTagHelperAttributeMethodName = "AddTagHelperAttribute";
             ExecutionContextAddHtmlAttributeMethodName = "AddHtmlAttribute";
             ExecutionContextOutputPropertyName = "Output";
-            StartWritingScopeMethodName = "StartWritingScope";
-            EndWritingScopeMethodName = "EndWritingScope";
+            StartTagHelperWritingScopeMethodName = "StartTagHelperWritingScope";
+            EndTagHelperWritingScopeMethodName = "EndTagHelperWritingScope";
             RunnerTypeName = "TagHelperRunner";
             ScopeManagerTypeName = "TagHelperScopeManager";
             ExecutionContextTypeName = "TagHelperExecutionContext";
             HtmlEncoderPropertyName = "HtmlEncoder";
+            TagHelperContentVariableName = "TagHelperContent";
```

Comment:
(hmm, browser search fail.  used just above.)
name needs to change - `TagHelperContentTypeName`